src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,77 @@
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  &__wrapper {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    @include stopwatch-size($stopwatch-size);
+    border: 2px solid #033c86;
+    border-radius: 50%;
+    background: radial-gradient(#ffff007b 5%, #007efd7b 80%);
+  }
+
+  &__second-hand {
+    position: absolute;
+    left: $stopwatch-size / 2 - $hand-width / 2;
+    top: 5%;
+    height: 45%;
+    width: $hand-width;
+    border-radius: 50% 50% 4px 4px;
+    background: linear-gradient(#ff0, #008efd);
```

Comment:
Don't repeat yourself, you have these rules also for minute-hand selector, so let's group them